UnitsNet/UnitDefinitions/LapseRate.json

```diff
@@ -1,14 +1,13 @@
 ﻿{
   "Name": "LapseRate",
-  "BaseUnit": "DegreeCelsiusPerMeter",
+  "BaseUnit": "DegreeCelsiusPerKilometer",
   "XmlDoc": "Lapse rate is the rate at which Earth's atmospheric temperature decreases with an increase in altitude, or increases with the decrease in altitude.",
   "Units": [
     {
-      "SingularName": "DegreeCelsiusPerMeter",
-      "PluralName": "DegreesCelciusPerMeter",
+      "SingularName": "DegreeCelsiusPerKilometer",
+      "PluralName": "DegreesCelciusPerKilometer",
       "FromUnitToBaseFunc": "x",
       "FromBaseToUnitFunc": "x",
-      "Prefixes": [ "Kilo" ],
```

Comment:
I assume you don't want DegreeCelsiusPerMeter anymore? It was included before in addition to the Kilo one.